addons/project/models/project.py

```diff
@@ -34,6 +34,7 @@
     'create_date',
     'write_date',
     'company_id',
+    'task_properties',
```

Comment:
Can you explain why it is needed? Normally the portal user cannot really add a property field and so it is not really needed for project sharing feature, no? 🤔

Reply:
Indeed it is not needed :+1:

Reply:
Ohhh okay, in fact, we have to change the search view used then. 🙂